### ISSUE TITLE
contentPadding value seems incorrect

### ISSUE THREAD

**Issue by solo5star** [yako-dev/flutter-settings-ui#54]
I found an error with your example, relevant with `contentPadding`.

```
════════ Exception caught by rendering library ═════════════════════════════════
The following assertion was thrown during performLayout():
Leading widget consumes entire tile width. Please use a sized widget, or consider replacing ListTile with a custom widget (see https://api.flutter.dev/flutter/material/ListTile-class.html#material.ListTile.4)
'package:flutter/src/material/list_tile.dart':
Failed assertion: line 1547 pos 7: 'tileWidth != leadingSize.width'
```

example code ([example/lib/screens/settings_screen.dart](https://github.com/yako-dev/flutter-settings-ui/blob/master/example/lib/screens/settings_screen.dart#L25)):
```dart
  Widget buildSettingsList() {
    return SettingsList(
      contentPadding: EdgeInsets.symmetric(horizontal: 180), // this line raises error
      sections: [
        SettingsSection(
          title: 'Common',
```

I think using `EdgeInsets.symmetric(vertical: 20)` would be correct instead of `EdgeInsets.symmetric(horizontal: 180)`.

here is the screenshot.

### Before
![image](https://user-images.githubusercontent.com/20203944/108152154-c3ebd380-711b-11eb-9ed0-5f3291a8f56d.png)

### After
![image](https://user-images.githubusercontent.com/20203944/108152182-d36b1c80-711b-11eb-934e-e95e933654e2.png)

<br>

It would be great if you consider correcting the example code.

Thanks.

**Comment by RusKryzhanovskiy** [comment]
Hi, @solo5star sorry for that!
It Will be fixed in `0.7.0`

**Comment by Androz2091** [comment]
It does not work for every sections
